src/test/java/org/broadinstitute/hellbender/tools/exome/PlotSegmentedCopyRatioIntegrationTest.java

```diff
@@ -53,8 +51,6 @@ public void testUnLoggedCommandLineSexChrs() throws IOException {
         final String sampleName = "HCC1143";
         File tDir = IOUtils.tempDir("Test", "Plotting");
         System.out.println(tDir);
```

Comment:
There is never a reason to have a system.out.println.  Use logging, instead (examples are found in other CLIs)